### DISCUSSION TITLE
Item Prices Tooltip: Add option to only show prices while holding alt

### DISCUSSION THREAD

**Discussion by ShaunDreclin** [runelite/runelite#15027] [Ideas]
Maybe have two options, one for bank and one for inventory.

**Comment by ShaunDreclin** [comment]
Also as a solution for #1075, have an option to show it while you're casting an alchemy spell.

**Comment by Nightfirecat** [comment]
We gate very few features on holding the "Alt" key, and I don't feel this should be one of them.

**Comment by ShaunDreclin** [comment]
Shift already changes the left click option on items for most people and I think even fewer things in the client use ctrl.
